src/resources/postcss/components/_search.pcss

```diff
@@ -112,7 +169,14 @@
 			@media (--viewport-medium) {
 				border: none;
 				border-right: 1px solid var(--color-border-default);
-				padding: 7px 18px 8px;
+			}
+		}
+	}
+
+	body.tribe-theme-twentynineteen & {
+		.tribe-common-c-search__input {
+			&:focus {
+				border-right-color: var(--color-border-default);
```

Comment:
this border color shouldn't be an issue if we move the border to the div wrapper